pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -979,23 +1012,34 @@ func (a *ApplicationBackupController) backupVolumes(backup *stork_api.Applicatio
 			drivers := a.getDriversForBackup(backup)
 			volumeInfos := make([]*stork_api.ApplicationBackupVolumeInfo, 0)
 			for driverName := range drivers {
-
 				driver, err := volume.Get(driverName)
 				if err != nil {
 					return err
 				}
-
+				// skip fetching status for skipped vols
+				if skipDriver == driverName {
```

Comment:
It is better to add the skip logic in the "getDriversForBackup" api itself, such it will not return the skipped drivers. 
I could see getDriversForBackup is called multiple places. We can avoid the check in all place.